src/main/java/games/strategy/triplea/delegate/BattleTracker.java

```diff
@@ -1135,6 +1137,36 @@ void fightAirRaidsAndStrategicBombing(final IDelegateBridge delegateBridge, Supp
     }
   }
 
+  /**
+   * 'Auto-fight' defenseless battles.
+   */
+  public void fightAutoKills(final IDelegateBridge bridge) {
```

Comment:
> Won't removing the javaDoc cause a checkstyle violation though?

Just to clarify the Checkstyle requirement...  The Google style guide requires Javadocs on all public methods that exceed two lines in length.